package.json

```diff
@@ -46,6 +46,10 @@
     "through2": "^2.0.0",
     "uglify-js": "^2.4.16"
   },
+  "devEngines": {
+    "node": "0.12 || 4.x || 5.x || 6.x || 7.x",
```

Comment:
Maybe once preset-env will support engines and we drop es2015 in favor of env. 😏